includes/class-wcpdf-settings-debug.php

```diff
@@ -36,120 +38,331 @@ public function debug_tools( $tab, $section ) {
+						<?php
+						if ( ! empty( $_POST ) && isset( $_POST['wpo_wcpdf_debug_tools_action'] ) && $_POST['wpo_wcpdf_debug_tools_action'] == 'delete_legacy_settings' ) {
+							// check permissions
+							if ( ! check_admin_referer( 'wpo_wcpdf_debug_tools_action', 'security' ) ) {
+								return;
+							}
+							// delete options
+							delete_option( 'wpo_wcpdf_general_settings' );
+							delete_option( 'wpo_wcpdf_template_settings' );
+							delete_option( 'wpo_wcpdf_debug_settings' );
+							// and delete cache of these options, just in case...
+							wp_cache_delete( 'wpo_wcpdf_general_settings','options' );
+							wp_cache_delete( 'wpo_wcpdf_template_settings','options' );
+							wp_cache_delete( 'wpo_wcpdf_debug_settings','options' );
 
-						printf('<div class="notice notice-success"><p>%s</p></div>', esc_html__( 'Legacy settings deleted!', 'woocommerce-pdf-invoices-packing-slips' ) );
-					}
-					?>
-				</form>
-			</p>
-			<p>
-				<a href="<?php echo esc_url( admin_url( 'admin.php?page=wpo-wcpdf-setup' ) ); ?>" class="button"><?php esc_html_e( 'Run the Setup Wizard', 'woocommerce-pdf-invoices-packing-slips' ); ?></a>
-			</p>
+							printf('<div class="notice notice-success"><p>%s</p></div>', esc_html__( 'Legacy settings deleted!', 'woocommerce-pdf-invoices-packing-slips' ) );
+						}
+						?>
+					</form>
+				</div>
+				<div class="tool">
+					<h4><?php _e( 'Run the Setup Wizard', 'woocommerce-pdf-invoices-packing-slips' ); ?></h4>
+					<p><?php _e( 'If you miss the initial configuration of this plugin when you installed it, you could do it again below.', 'woocommerce-pdf-invoices-packing-slips' ); ?></p>
+					<a href="<?php echo esc_url( admin_url( 'admin.php?page=wpo-wcpdf-setup' ) ); ?>" class="button"><?php esc_html_e( 'Run the Setup Wizard', 'woocommerce-pdf-invoices-packing-slips' ); ?></a>
+				</div>
+				<div class="tool">
+					<h4><span><?php _e( 'Export Settings', 'woocommerce-pdf-invoices-packing-slips' ); ?></span></h4>
+					<p><?php _e( 'Download the settings of the plugin for this website as a .json file. This allows for effortless importation of the configuration into another site.', 'woocommerce-pdf-invoices-packing-slips' ); ?></p>
+					<form class="wpo_wcpdf_debug_tools_form" method="post">
+						<input type="hidden" name="debug_tool" value="export-settings">
+						<fieldset>
+							<select name="type" required>
+								<?php
+									foreach ( $this->get_setting_types() as $type => $name ) {
+										?>
+										<option value="<?= $type; ?>"><?php printf( /* translators: type of settings */ __( "%s Settings", 'woocommerce-pdf-invoices-packing-slips' ), $name ); ?></option>
+										<?php
+									}
+								?>
+							</select>
+							<a href="" class="button button-secondary submit"><?php _e( 'Export', 'woocommerce-pdf-invoices-packing-slips' ); ?></a>
```

Comment:
When clicking on the "Export" button, I understood that you then have to click on the link to download the JSON file. However, I think we could add the word "Download" (or something similar) just before the link, which would make even more explicit the next action the user has to do. 

![image](https://user-images.githubusercontent.com/38109855/216471500-3818665a-6af1-4514-a92c-676b8dc25721.png)

Another better approach, it would be to fire this action automatically, i.e. download the file right after clicking the button, to save the user a step. I understand that it is easier to say than to make, but it could be done in the future, if it needs to switch the current approach you used to generate the file, or if it would take too much time.